test/Fixtures.purs

```diff
@@ -910,6 +926,96 @@ seabugMetadataDeltaFixture1 = SeabugMetadataDelta
       10
   }
 
+cip25MetadataFilesFixture1 :: Array CIP25MetadataFile
+cip25MetadataFilesFixture1 = CIP25MetadataFile <$>
+  [ { name: unsafePartial $ fromJust $ byteArrayFromString "file_name_1"
+    , mediaType: unsafePartial $ fromJust $ byteArrayFromString "media_type"
+    , uris: unsafePartial $ fromJust $ traverse byteArrayFromString $
+        "uri1" :| [ "uri2", "uri3" ]
+    }
+  , { name: unsafePartial $ fromJust $ byteArrayFromString "file_name_2"
+    , mediaType: unsafePartial $ fromJust $ byteArrayFromString "media_type_2"
+    , uris: unsafePartial $ fromJust $ traverse byteArrayFromString $
+        "uri4" :| [ "uri5", "uri6" ]
+    }
+  ]
+
+cip25MetadataEntryFixture1 :: CIP25MetadataEntry
+cip25MetadataEntryFixture1 = CIP25MetadataEntry
+  { policyId: policyId
+  , assetName: tokenName1
+  , imageUris: unsafePartial $ fromJust $ traverse byteArrayFromString $
+      "image_uri1" :| [ "image_uri2", "image_uri3" ]
+  , mediaType: byteArrayFromString "media_type"
+  , description: unsafePartial $ fromJust $ traverse byteArrayFromString $
+      [ "desc1", "desc2", "desc3" ]
+  , files: cip25MetadataFilesFixture1
+  }
+
+cip25MetadataEntryFixture2 :: CIP25MetadataEntry
+cip25MetadataEntryFixture2 = CIP25MetadataEntry
+  { policyId: policyId
+  , assetName: tokenName2
+  , imageUris: unsafePartial $ fromJust $ traverse byteArrayFromString $
+      "image_uri1" :| []
+  , mediaType: Nothing
+  , description: []
+  , files: []
+  }
+
+cip25MetadataFixture1 :: CIP25Metadata
+cip25MetadataFixture1 = CIP25Metadata
+  [ cip25MetadataEntryFixture1, cip25MetadataEntryFixture2 ]
+
+cip25MetadataJsonFixture1 :: Json.Json
+cip25MetadataJsonFixture1 = fromRight Json.jsonNull $ Json.parseJson $
+  """
```

Comment:
I think for small JSON strings it's OK to use literals, but for larger fixtures like this I'd rather put it into a file. It would be easier to spot any accidental issues with the JSON since editors would highlight them